cypress/support/cypress-commands/commands.js

```diff
@@ -1549,3 +1548,34 @@ Cypress.Commands.add('startOrStopInteractionsService', (action) => {
 Cypress.Commands.add('envConfigSetup', () => {
   fireLog.info('No additional config module environment setup');
 });
+
+/**
+ * @module commands
+ * @function initiatePerformanceMetrics
+ * @description Creates a marker and saves the start time in THRESHOLD_MONITOR_START_TIME env, if performance metrics is enabled.
+ * @example
+ * cy.initiatePerformanceMetrics()
+ */
+Cypress.Commands.add('initiatePerformanceMetrics', () => {
+  // Retrieve the scenario name from the env.
+  const scenarioName = Cypress.env(CONSTANTS.SCENARIO_NAME);
+
+  // Check if performance metrics is enabled
+  if (UTILS.getEnvVariable(CONSTANTS.PERFORMANCE_METRICS) === true) {
+    // Request to create a marker
+    const requestMap = {
+      method: CONSTANTS.REQUEST_OVERRIDE_CALLS.CREATE_MARKER,
+      params: scenarioName,
+    };
+    fireLog.info(`Firebolt Call to 1st party App: ${JSON.stringify(requestMap)} `);
+    cy.sendMessagetoPlatforms(requestMap).then((result) => {
+      fireLog.isTrue(result.success, 'Response for marker creation: ' + JSON.stringify(result));
+    });
+
```

Comment:
logs 1570 and 1572 to be removed before merge. the log issue will be taken in  https://ccp.sys.comcast.net/browse/FIRECERT-2480